modules/nw-ovn-k-baseline-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: Concept
+[id="BaselineAdminNetworkPolicy"_{context}"]
+= BaselineAdminNetworkPolicy
+
+`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definitions (CRDs). As a {product-title} administrator, you can use BANP to setup and enforce optional baseline network policies that are overridable by users using `NetworkPolicies` if need be. Rule actions for BANP are `allow` or `deny`.
+
+BANP allows administrators to specify:
+
+* A `subject` that consists of a set of pods that is selected by the policy.
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
+
+[NOTE]
+====
+`BaselineAdminNetworkPolicy` is a `TechnologyPreviewNoUpgrade` feature that can be enabled on test clusters that are not in production. The feature is behind the flag `enable-admin-network-policy`.
+====
+
+[discrete]
+[id="BaselineAdminNetworkPolicy-example"]
+== BaselineAdminNetworkPolicy example
+
+The `BaselineAdminNetworkPolicy` resource is a cluster singleton `object` that can be used as a guardrail policy incase a passed traffic policy does not match any `NetworkPolicies` in the cluster. BANP can also be used as a default security model that provides guardrails that intra-cluster traffic is blocked by default and user will need to use `NetworkPolicy` to allow known traffic. You must use `default` as the name when creating a `BaselineAdminNetworkPolicy` object.
+
+
+.Example YAML file for BANP
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: BANP_example <1>
```

Comment:
this is wrong, you can't have any other name other than `default` which brings me to my next question @JoeAldinger have you tested all the yamls on a live cluster? Users won't like it if they copy this and it doesn't work.

Reply:
also just noticed this is still a ANP example?

Reply:
and there is no pass in BANP, the rule name suggests otherwise